acceptance/acceptance_test.go

```diff
@@ -399,30 +399,14 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	defer cancelFunc()
 	args := []string{"bash", "-euo", "pipefail", EntryPointScript}
 	cmd := exec.CommandContext(ctx, args[0], args[1:]...)
-	cmd.Env = os.Environ()
-	cmd.Env = append(cmd.Env, "UNIQUE_NAME="+uniqueName)
-	cmd.Env = append(cmd.Env, "TEST_TMP_DIR="+tmpDir)
-
-	workspaceClient := internal.PrepareServerAndClient(t, config, LogRequests, tmpDir)
 
-	// Configure resolved credentials in the environment.
-	cmd.Env = append(cmd.Env, "DATABRICKS_HOST="+workspaceClient.Config.Host)
-	if workspaceClient.Config.Token != "" {
-		cmd.Env = append(cmd.Env, "DATABRICKS_TOKEN="+workspaceClient.Config.Token)
-	}
-
-	var user iam.User
-	if isRunningOnCloud {
-		pUser, err := workspaceClient.CurrentUser.Me(context.Background())
-		require.NoError(t, err, "Failed to get current user")
-		user = *pUser
-	} else {
-		// For the purposes of replacements, use testUser for local runs.
-		// Note, users might have overriden /api/2.0/preview/scim/v2/Me but that should not affect the replacement:
-		user = internal.TestUser
-	}
+	cfg, user := internal.PrepareServerAndClient(t, config, LogRequests, tmpDir)
 	testdiff.PrepareReplacementsUser(t, &repls, user)
-	testdiff.PrepareReplacementsWorkspaceClient(t, &repls, workspaceClient)
+	testdiff.PrepareReplacementsWorkspaceConfig(t, &repls, cfg)
+
+	cmd.Env = auth.ProcessEnv(cfg)
```

Comment:
not blocking for this PR, but can we come up with a descriptive name for this function? ResetAuthEnvVars? UnsetAndSetAuthEnvVars?

Reply:
Is `ProcessEnv` not descriptive enough? This function does not set or unset environment variables; it just provides a list of env vars to use.

Reply:
that would be better than ProcessEnv.